lib/jira/commands/tickets.rb

```diff
@@ -1,43 +1,52 @@
 module Jira
   class CLI < Thor
 
-    desc "tickets", "List the tickets of the input username"
+    desc "tickets [--jql]", "List the tickets of the input username (or jql)"
```

Comment:
Since when we specify JQL, the input username is being ignored, I feel like it might make sense to have the default to this command be JQL.

In other words:
```
def tickets(jql="assignee=#{Jira::Core.username}")
  Command::Tickets.new(jql).run
end
```

I think this would simplify the interface and we wouldn't have any additional switching logic for when the `--jql` option is specified. Thoughts?